tasks-schema.json

```diff
@@ -311,12 +413,19 @@
                       ]
                     },
                     "cdf": {
+                      "description": "Object defining the cumulative distribution function of the predictive distribution output type.",
                       "type": "object",
                       "properties": {
                         "type_id": {
+                          "description": "Object containing required and optional arrays defining possible values of the target variable at which values of the cumulative distribution function of the predictive distribution will be recorded. Values are therefore target dependent.",
```

Comment:
Does values being "target dependent" mean that the targets defined in this set of task_ids must all (if there are more than 1) share the same cdf values? That might be worth noting somewhere else  in the documentation.